### PR TITLE
Adding PostgreSQL

### PR DESCRIPTION
**Pull request by matisku**

### CODE REVIEW

File: templates/postgres/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      description: "Postgres Password"
+      label: "Postgres Password"
+      required: true
+      default: "postgres_password"
+      type: "string"
+      
+version: '2'
+services:
+  postgres-lb:
+    scale: 1
```
Comment (rawmind0) [inline templates/postgres/0/rancher-compose.yml]: Add lb config to config lb as protocol: tcp
```
    lb_config:
      certs: []
      port_rules:
      - protocol: tcp
        service: postgres
        source_port: ${lb_port}
        target_port: 5432
```

**Comment by rawmind0** [comment]
Hi @matisku ... Thank for the PR and to contribute...

Great adding to have postgres package!!

I've deployed it with a lb issue. It's configurated as http protocol, and it should be tcp. I make you a change request to fix it.

Best regards...

File: templates/postgres/0/rancher-compose.yml (hunk 1 above)
Comment (matisku) [inline templates/postgres/0/rancher-compose.yml]: Hello,

Thanks for a suggestion. I added requested changes.

**Comment by rawmind0** [comment]
Thanks @matisku ....

LGTM....

**Pull request merged**